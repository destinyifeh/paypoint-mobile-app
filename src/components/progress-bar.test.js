'use strict';

import React from 'react';
import { configure } from 'enzyme';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';

import ProgressBar from './progress-bar';

configure({
  adapter: new Adapter()
});

it('renders correctly', () => {
  const tree = renderer
    .create(<ProgressBar />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
