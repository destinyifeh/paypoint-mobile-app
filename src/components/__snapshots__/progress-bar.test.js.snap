// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#32BE69",
      "height": 10,
      "width": "20%",
    }
  }
/>
`;
